tmgmt_thebigword.connector.inc

```diff
@@ -150,6 +150,17 @@ class TheBigWordConnector {
       $response = drupal_http_request($url, $options);
     }
     catch (HttpRequestException $e) {
```

Comment:
drupal_http_request() doesn't throw exceptions, this doesn't work. You need to check the $response object for an error instead, see https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_http_request/7.x